packages/dtc/makefile.patch

```diff
@@ -0,0 +1,18 @@
+--- dtc-1.4.7/Makefile	2018-12-24 00:56:22.322225248 +0530
++++ dtc-1.4.7.mod/Makefile	2018-12-24 00:57:22.645564591 +0530
+@@ -25,13 +25,13 @@
+ SWIG = swig
+ PKG_CONFIG ?= pkg-config
+ 
+-INSTALL = /usr/bin/install
++INSTALL = @TERMUX_PREFIX@/bin/install
+ INSTALL_PROGRAM = $(INSTALL)
+ INSTALL_LIB = $(INSTALL)
+ INSTALL_DATA = $(INSTALL) -m 644
+ INSTALL_SCRIPT = $(INSTALL)
+ DESTDIR =
+-PREFIX = $(HOME)
++PREFIX = @TERMUX_PREFIX@
```

Comment:
This can be placed to build.sh. Patch will be unneeded at all.

```
TERMUX_PKG_EXTRA_MAKE_ARGS="PREFIX=$TERMUX_PREFIX"
```

Reply:
Does same but ok I'll remove the patch